lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,8 @@
+namespace :heart_rate_data do
+  desc "Update heart rate data for each user for Fitbit"
+  task sync: :environment do
+    HeartRateData.all.each do |heart_rate_data|
+      heart_rate_data.update_heart_rate_data!
+    end
```

Comment:
The last user happens to be me. It's a kludge, but it'll get the app in a workable state.

You should keep the one-to-many relationship. A new record will be created each day so I expect that user_id 3 will have 365 heart_rate_datas records a year.

RE refreshing the API token: great!